pom.xml

```diff
@@ -23,7 +23,7 @@
 
     <groupId>com.github.cafapi</groupId>
     <artifactId>opensuse-base-image</artifactId>
-    <version>1.2.0-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
As discussed, there doesn't seem to be a breaking change here (so far as we can see).